lib/ruy/conditions/between.rb

```diff
@@ -1,28 +1,27 @@
 module Ruy
   module Conditions
 
-    # Expects that a context value belongs to a given range.
+    # Expects that a context key stores a value between a defined range
     #
-    # Comparison formula: from <= context[attr] <= to
     class Between < Condition
-      attr_reader :from, :to, :range, :attr
+      attr_reader :from, :to, :range, :key
 
-      # @overload initialize(from, to, attr)
+      # @overload initialize(from, to, key)
       #   @param from Range lower bound
       #   @param to Range upper bound
-      #   @param attr Name of the attribute that will be evaluated
-      # @overload initialize(range, attr)
+      #   @param key Name of the attribute that will be evaluated
```

Comment:
Shouldn't be attribute.
 
Alternative:
Name of the key for which its value will be evaluated